src/components/Table.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="board" id="board" >
+    <div class="matrix" id="create-board">
+      
+    </div>
+  </div>
+</template>
+
+<script>
+import {EventBus} from '@/services/event-bus';
+export default {
+  name: 'board',
+  methods: {
+    drawBoard() {
+       var codigoHTML="<table border=\"1\"><tbody>";
+        for (var i = 0; i < this.rows; i++) {
+          codigoHTML+="<tr>";
+          for (var j = 0; j < this.columns; j++) {
+            codigoHTML+="<td></td>";
+          }
+          codigoHTML+="</tr>";
+        }
+        codigoHTML+="</tbody></table>";
+        document.getElementById("create-board").innerHTML = codigoHTML;
+      }
+  },
+  data() {
+    return {
+      rows: '',
+      columns : '',
+    }
+  },
+  mounted() {
+    EventBus.$on('dataBoard',(row, col) => {
```

Comment:
event names should be descriptive and should represent an action, like "size-change", "heigh-change"m "width-change", etc

Reply:
rename the event